src/app/modules/group/http-services/get-group-managers.service.ts

```diff
@@ -7,7 +7,7 @@ export interface Manager {
   id: string,
   name: string,
 
-  can_manage: string,
+  can_manage: 'none' | 'memberships' | 'memberships_and_group',
```

Comment:
I now restrict the type of `can_manage` to only possible values instead of just a `string`